server/models/post.js

```diff
@@ -0,0 +1,37 @@
+const db = require('../dbConfig/init');
+const { v4: uuidv4 } = require('uuid');
+
+module.exports = class Post {
```

Comment:
Module.exports here? I dunno if that's weird or just me but again just trying to find things to criticise

Reply:
That is how the cool kids do it now